distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSingleSignOn.java

```diff
@@ -329,9 +358,9 @@ public void testGuestAuth() throws Exception {
                 statusCode(200).
         when().
                 get(searchHelper.redirectUrl);
+        // @formatter:on
     }
 
-    @Ignore
     @Test
```

Comment:
Thanks for removing the @Ignore! I verified that the changes we added in the previous commit fixed its intermittent failure.